.ci/macos-steps.yaml

```diff
@@ -17,6 +17,7 @@ steps:
     pip install cython numpy pandas zipp
     brew update
     brew install openblas armadillo boost
+    brew upgrade hdf5
```

Comment:
```suggestion
    brew install openblas boost
    brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/9620946e004a29744f9c5e4a27800cba6d6d6b9b/Formula/armadillo.rb
```

Reply:
Hi @zoq, So I tried this this version and even some earlier version (I think till 9.800.4) and they gave me another build error same as the one on the pipeline i.e. pp:26:14:
```
 fatal error: '/usr/local/Cellar/hdf5/1.10.6/include/hdf5.h' file not found
    #include ARMA_INCFILE_WRAP(ARMA_HDF5_HEADER)
```
The only version that didn't give this error was 9.850.1_1 (current) but that fails load and save test. The test fails for all hd5 formats i.e. h5, hdf5 and so on.